homeassistant/components/harmony/strings.json

```
@@ -53,7 +53,7 @@
     },
     "change_channel": {
       "name": "Change channel",
-      "description": "Sends change channel command to the Harmony HUB.",
+      "description": "Sends the change channel command to the Harmony Hub.",
```

This is more natural: 

```suggestion
      "description": "Sends a change channel command to the Harmony Hub.",
```